build.sh

```diff
@@ -12,7 +12,8 @@ find package -name "*.ts" -delete
 find package -name "*.map" -delete
 
 rm -rf package/build/src/{events,load-balancer-eds,object-stream,xds-client,xds-bootstrap}.js
-replace "const GoogleAuth = require('google-auth-library')" "throw new Error('unimplemented');" "  .GoogleAuth;" "" -- package/build/src/channel-credentials.js
+sed -i "/const GoogleAuth = require('google-auth-library')/d" package/build/src/channel-credentials.js
+sed -i "s/    \.GoogleAuth/throw new Error('unimplemented')/" package/build/src/channel-credentials.js
```

Comment:
```suggestion
sed -i "s/    \.GoogleAuth;/throw new Error('unimplemented');/" package/build/src/channel-credentials.js
```

Let's be sure it doesn't affect e.g. `.GoogleAuthWhatever`.